cmd/agent/main.go

```diff
@@ -18,20 +18,45 @@ import (
 	"github.com/npavlov/go-metrics-service/internal/utils"
 )
 
+var (
+	buildVersion = "N/A"
```

Comment:
You can move it to the suitable package. For example, `version`.

Reply:
fixed